cmd/snap-confine/mount-support.c

```diff
@@ -878,6 +903,41 @@ static bool __attribute__((used))
 	return false;
 }
 
+static struct sc_mount *sc_create_homedir_mounts(const struct sc_invocation *inv)
```

Comment:
it seems create is used mostly for things happening to the system but this is only setting things, so mabye s/create/init/ ?

Reply:
Renamed this, thank you for the review